docs/pages/docs/guides/custom-admin-ui-components.mdx

```diff
@@ -0,0 +1,45 @@
+import { ComingSoon } from '../../../components/docs/ComingSoon';
+import { Markdown } from '../../../components/Markdown';
+
+# Custom Admin UI Pages
+
+In this guide we'll show you how to add custom components to the Keystone Admin UI.
+
+To import a custom component for rendering within the admin-ui, ensure that the `admin/config` file exists in the root of your Keystone Project.
+This file expects a named export `components`. **Presently this `components` export only supports a custom Logo component**:
+
+```tsx
+// admin/config.tsx
+function CustomLogo () {
+    return <h3>Custom Logo here </h3>
+}
+
+export const components = {
+    Logo: CustomLogo
+}
+```
+
+If you have styling constraints, we recommend using the jsx export from the `@keystone-ui/core` package, as this will ensure that the version of emotion you're using conforms with the version of emotion used internally within Keystone.
+
+```tsx
+// admin/config.tsx
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@keystone-ui/core';
+
+function CustomLogo () {
+    return <h3 css={{
+        background: 'papayawhip'
+    }}>Custom Logo here </h3>
+}
+
+export const components = {
+    Logo: CustomLogo
+}
+```
+
+Of course this is purely a recommendation, if you would prefer to roll your own css-in-js solution in with your custom component please feel free to! Although this may require additional configuration outside of the scope of this guide.
+
+?> **Old JSX transform** As of the time of writing this guide, Keystone is internally still leveraging the old JSX transform, which means that **within the keystone repo** any `.tsx` files will need to be preceded with a React import (or have React in scope). This is why there are `import React from 'react'` statements in our examples, this is NOT necessary outside of the Keystone repo.
```

Comment:
"Keystone is internally still leveraging the old JSX transform" feels strange and untrue to me. some meh words: "in the keystone repo, we happen to have a babel config that means we use the old jsx transform(this doesn't have an impact on the code we ship to npm, it's all compiled by that point and the fact that we use the old jsx transform isn't really in _Keystone_ itself, only the _repo_). unless you have a babel config with the old jsx transform(which is currently the default with @babel/preset-react), you'll be using Next's babel config which uses the new jsx transform

Reply:
How's this @mitchellhamilton 
>The Keystone monorepo leverages a babel config that means we use the old jsx transform (this doesn't have an impact on the code we ship to npm). This is why there are `import React from 'react'` statements in our examples, this is NOT necessary outside of the Keystone repo (unless you have a babel config with the old jsx transform which is currently the default with @babel/preset-react) as you'll be using Next's babel config which uses the new jsx transform